dockerfiles/theia/e2e/cypress/integration/theia/typescript.spec.ts

```diff
@@ -65,6 +65,9 @@ context('TypeScript', () => {
                     });
                 })
             });
+        }).then(() => {
+            // enable the explorer view
+            cy.get('body').type('{ctrl}{cmd}{shift}e')
```

Comment:
was this behavior changed upstream?

Reply:
@azatsarynnyy I suppose as it was always open before :-/